docs/api.md

```diff
@@ -74,11 +74,11 @@ module.exports =
   })
 ```
 
-?> **Notice the idempotence check in the handler.**  We only want to act on a message if we haven't previously acted on it.  Messages in a stream are processed in order, but consumers restart all the time (whether by chaos or actual deploys), so messages are frequently reprocessed.  Therefore, your message handlers **must be idempotent.**
+?> **Notice the idempotence check in the handler.**  We only want to act on a message if we haven't previously acted on it.  Messages in a stream are processed in order, but consumers restart all the time (whether by chaos or actual deploys), so messages are frequently reprocessed.  Therefore, your message handlers **must be idempotent.**  A collection of common idempotence patterns is included in [Best Practices](/best-practices?id=idempotence-patterns).
 
 ### Starting / Stopping
 
-A consumer is constructed in a dormant state, and will not begin processing messages until you call its `start()` function.  If desired, multiple consumers can be run together on a single host.  Starting them together is simple:
+A consumer is constructed in a dormant state, and will not begin processing messages until you call its `start()` function.  If desired, multiple related consumers that together form one [component]() can be run together on a single host.  Starting them together is simple:
```

Comment:
Forget a link? Or am i bad at markdown?

Reply:
Not forgotten, more of a placeholder.  I haven't written the `Event Sourcing` page yet, but I knew I'd want to link to things in it.  Figured I could search `]()` to find the empty links and backfill them later.

I could stub out the headings and link them early, but I was worried they might change and I'd forget to fix the links.